leasing/migrations/0076_paikkatietovipunen_vuokraalueet_view_vuokraustunnus_and_tyypin_tunnus.py

```diff
@@ -0,0 +1,27 @@
+from pathlib import Path
+
+from django.db import migrations
+
+
+def load_sql_file(name):
+    file_path = Path(__file__).resolve().parent / name
+    with open(file_path, encoding="utf-8") as file:
+        # The necessary DROP VIEW added to avoid editing the reverse sql file
+        return "DROP VIEW public.paikkatietovipunen_vuokraalueet;" + file.read()
```

Comment:
This feels like the wrong place to hide this detail in.
The function indicates it loads sql file, not that the loaded file would have some inplace additions.

Reply:
Done